proto/quary/service/v1/chart.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package quary.service.v1;
+
+option java_multiple_files = true;
+
+message Chart {
+  string name = 1;
+  optional string description = 2;
+  repeated string tags = 3;
+
+  oneof dependency {
+    ChartDependency path = 5;
```

Comment:
nit: we use 'path' for fully qualified path in other places might be confusing convention